tests/stp/stripe/run.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+cd $HOME/TorQ/deploy/TorQ/latest
+source setenv.sh
+
+# Handle command-line arguments
+source $KDBTESTS/flagparse.sh
+
+# Path to test directory
+testpath=${KDBTESTS}/stp/stripe
+export perfpath=${testpath}/performance
+
+# Start procs
+${TORQHOME}/torq.sh start discovery1 stp1 rdb1 rdb2 rdb3 rdb4 -csv ${testpath}/process.csv
+
+# Start test proc
+QCMD='taskset -c 0,1 /usr/bin/rlwrap q'
+${QCMD} ${TORQHOME}/torq.q \
+  -proctype test -procname test1 \
+  -test ${testpath} \
+  -load ${KDBTESTS}/helperfunctions.q ${testpath}/settings.q \
+  -testresults ${KDBTESTS}/stp/results/stripe/ \
+  -runtime $run \
+  -procfile ${testpath}/process.csv \
+  -write
+  #$debug $stop $write $quiet
+
+# k4unit test
+k4unit=$(find "$(cd ..; pwd)" -name "k4unit.q")
+# Performance test from stp
+cd $perfpath
+# Specify session name
+session=kdbProcess
+# Kill remaining session first
+tmux kill-session -t $session
+# Start a new session (kdbProcess) detached (-d)
+tmux new-session -d -s $session
+# Wait till session and window is created
+while ! tmux has-session -t $session:bash; do sleep 1; done
+# qcon into stp proc
+stpport=$(cat ${testpath}/process.csv | grep segmentedtickerplant | awk -F',' '{print $2}')
+tmux send-keys -t kdbProcess:bash -l qcon=\'/usr/bin/rlwrap\ /opt/kdb/qcon\'
+tmux send-keys -t kdbProcess:bash Enter
+tmux send-keys -t kdbProcess:bash stpport=\'$stpport\' Enter
+tmux send-keys -t kdbProcess:bash -l '$qcon homer.aquaq.co.uk:$stpport:admin:admin'
+tmux send-keys -t kdbProcess:bash Enter
+# Load the script file
+tmux send-keys -t kdbProcess:bash \\l\ $k4unit Enter
+# Load settings
+settings=$testpath/settings.q
+tmux send-keys -t kdbProcess:bash \\l\ $settings Enter
+# Run performance test and save results
+tmux send-keys -t kdbProcess:bash \\l\ $perfpath/run_perf_test.q Enter
```

Comment:
I'm a bit confused about all this stuff, why can't we just run the TorQ process with the `-test {whatever}` arg and let TorQ load k4unit etc. automatically?

Reply:
This is related to the problem above. Because the `-test` did not work as I expected, I had to use the run script to 

- qcon into the stp
- load the specific k4unit test files etc. 
- run the test

which is why I relied on tmux to "send the keys".

I tried to minimize the loading of the files by passing the k4unit.q, settings.q etc into the **load** column but it breaks the stp so I had to load it after it starts up.